jangjuseong/app.js

```diff
@@ -194,6 +199,31 @@ app.post('/likes/:userId/:postId', async (req, res) => {
   return res.status(201).json({ message: 'likeCreated' });
 });
 
+app.get('/users/signup', async (req, res) => {
+  const { email, password } = req.body;
+
+  const [userData] = await mysqlDataSource.query(
+    `
+    SELECT * 
+    FROM users
+    WHERE email=?
+    `,
+    [email]
+  );
+
+  const payLoad = { email: userData.email };
+  const secretKey = 'mySecretKey';
```

Comment:
secretKey 역시 토큰을 검증하는데 있어서 중요한 역할을 합니다.
토큰을 암호화하고 복호화하는 과정에서 해당 토큰이 백엔드 서버의 secretKey를 기반으로 암호화 되었는지 확인하기 때문입니다.
이 값이 노출될 경우 누군가 악의적으로 토큰을 만들어낼 수 있으므로 환경변수에서 관리할 수 있도록 수정해주세요!

```suggestion
  const SECRET_KEY = process.env.SECRET_KEY;
```